mmv1/products/certificatemanager/api.yaml

```diff
@@ -212,4 +212,207 @@ objects:
             description: |
               Authorizations that will be used for performing domain authorization
             item_type: Api::Type::String
+  - !ruby/object:Api::Resource
+    name: 'CertificateMap'
+    base_url: 'projects/{{project}}/locations/global/certificateMaps'
+    create_url: 'projects/{{project}}/locations/global/certificateMaps?certificateMapId={{name}}'
+    self_link: 'projects/{{project}}/locations/global/certificateMaps/{{name}}'
+    update_verb: :PATCH
+    update_mask: true
+    description: |
+      CertificateMap defines a collection of certificate configurations,
+      which are usable by any associated target proxies
+    async: !ruby/object:Api::OpAsync
+      operation: !ruby/object:Api::OpAsync::Operation
+        path: 'name'
+        base_url: '{{op_id}}'
+        wait_ms: 1000
+      result: !ruby/object:Api::OpAsync::Result
+        path: 'response'
+      status: !ruby/object:Api::OpAsync::Status
+        path: 'done'
+        complete: true
+        allowed:
+          - true
+          - false
+      error: !ruby/object:Api::OpAsync::Error
+        path: 'error'
+        message: 'message'
+    parameters:
+      - !ruby/object:Api::Type::String
+        name: 'name'
+        required: true
+        input: true
+        url_param_only: true
+        pattern: projects/{{project}}/locations/global/certificateMaps/{{name}}
+        description: |
+          A user-defined name of the Certificate Map. Certificate Map names must be unique
+          globally and match the pattern `projects/*/locations/*/certificateMaps/*`.
+    properties:
+      - !ruby/object:Api::Type::String
+        name: 'description'
+        description: |
+          A human-readable description of the resource.
```

Comment:
Do we want to add `One or more paragraphs of text description of a certificate map.` to be consistent with the API docs?

Reply:
@sheneska would you mind doing this too? Thanks!